src/onediff/pipeline_stable_diffusion_img2img_oneflow.py

```diff
@@ -610,9 +610,9 @@ def __call__(
             list of `bool`s denoting whether the corresponding generated image likely represents "not-safe-for-work"
             (nsfw) content, according to the `safety_checker`.
         """
-        message = "Please use `image` instead of `init_image`."
-        init_image = deprecate("init_image", "0.13.0", message, take_from=kwargs)
-        image = init_image or image
+        # message = "Please use `image` instead of `init_image`."
+        # init_image = deprecate("init_image", "0.13.0", message, take_from=kwargs)
+        # image = init_image or image
```

Comment:
这里为什么注释

Reply:
```
╭───────────────────────────── Traceback (most recent call last) ──────────────────────────────╮
│ /data/home/baijingwen/dev/diffusers/examples/image_to_image.py:21 in <module>                │
│                                                                                              │
│   18 img = Image.new("RGB", (512, 512), "#1f80f0")                                           │
│   19                                                                                         │
│   20 with flow.autocast("cuda"):                                                             │
│ ❱ 21 │   images = pipe(                                                                      │
│   22 │   │   prompt,                                                                         │
│   23 │   │   image=img,                                                                      │
│   24 │   │   guidance_scale=10,                                                              │
│                                                                                              │
│ /data/home/baijingwen/anaconda3/envs/diff/lib/python3.10/site-packages/oneflow/autograd/auto │
│ grad_mode.py:154 in wrapper                                                                  │
│                                                                                              │
│   151 │   def __call__(self, func):                                                          │
│   152 │   │   def wrapper(*args, **kwargs):                                                  │
│   153 │   │   │   with AutoGradMode(False):                                                  │
│ ❱ 154 │   │   │   │   return func(*args, **kwargs)                                           │
│   155 │   │                                                                                  │
│   156 │   │   return wrapper                                                                 │
│   157                                                                                        │
│                                                                                              │
│ /data/home/baijingwen/anaconda3/envs/diff/lib/python3.10/site-packages/onediff/pipeline_stab │
│ le_diffusion_img2img_oneflow.py:614 in __call__                                              │
│                                                                                              │
│   611 │   │   │   (nsfw) content, according to the `safety_checker`.                         │
│   612 │   │   """                                                                            │
│   613 │   │   message = "Please use `image` instead of `init_image`."                        │
│ ❱ 614 │   │   init_image = deprecate("init_image", "0.13.0", message, take_from=kwargs)      │
│   615 │   │   image = init_image or image                                                    │
│   616 │   │                                                                                  │
│   617 │   │   # 1. Check inputs                                                              │
│                                                                                              │
│ /data/home/baijingwen/anaconda3/envs/diff/lib/python3.10/site-packages/diffusers/utils/depre │
│ cation_utils.py:18 in deprecate                                                              │
│                                                                                              │
│   15 │                                                                                       │
│   16 │   for attribute, version_name, message in args:                                       │
│   17 │   │   if version.parse(version.parse(__version__).base_version) >= version.parse(vers │
│ ❱ 18 │   │   │   raise ValueError(                                                           │
│   19 │   │   │   │   f"The deprecation tuple {(attribute, version_name, message)} should be  │
│   20 │   │   │   │   f" version {__version__} is >= {version_name}"                          │
│   21 │   │   │   )                                                                           │
╰──────────────────────────────────────────────────────────────────────────────────────────────╯
ValueError: The deprecation tuple ('init_image', '0.13.0', 'Please use `image` instead of 
`init_image`.') should be removed since diffusers' version 0.14.0 is >= 0.13.0
```
报错说升级diffusers的话应该去掉，我就加了个注释，如果可以去的话就把它删除。